src/PhenX.EntityFrameworkCore.BulkInsert.MySql/MySqlBulkInsertProvider.cs

```diff
@@ -26,6 +26,9 @@ public MySqlBulkInsertProvider(ILogger<MySqlBulkInsertProvider>? logger = null)
     /// <inheritdoc />
     protected override string GetTempTableName(string tableName) => $"#_temp_bulk_insert_{tableName}";
 
+    /// <inheritdoc />
+    protected override MySqlBulkInsertOptions GetDefaultOptions() => new();
```

Comment:
I would call it "CreateDefaultOptions()" because you cann reuse them.

You can also just define that as `TOptions : class, new()` and handle it the base provider.

Reply:
I'll rename it, but will not change the constraint